src/code/z_en_item00.c

```diff
@@ -1037,37 +1037,37 @@ void Item_DropCollectibleRandom(GlobalContext* globalCtx, Actor* fromActor, Vec3
         }
 
         if (dropId == ITEM00_FLEXIBLE) {
-            if (gSaveContext.perm.unk24.currentLife < 0x11) {
+            if (gSaveContext.health < 0x11) {
                 Actor_Spawn(&globalCtx->actorCtx, globalCtx, ACTOR_EN_ELF, spawnPos->x, spawnPos->y + 40.0f, spawnPos->z, 0, 0, 0, 2);
                 func_800F0568(globalCtx, spawnPos, 0x28, 0x28E7);
                 return;
             }
 
-            if (gSaveContext.perm.unk24.currentLife < 0x31) {
+            if (gSaveContext.health < 0x31) {
```

Comment:
Health should be decimal IMO.

Reply:
i don't see why 🤔 really there should just be a macro for health

Reply:
Health should definitely be a decimal.

Reply:
"should definitely be decimal" when the values dont make sense to begin with, a single heart is a value of 10, so neither hex or decimal is going to make sense until there is a macro for health

Reply:
If a heart is a multiple of ten, then we should always have health in decimal imo.

Reply:
Is this even my change to make though? Doesn't really have much to do with save context as it was this way already.

Reply:
No, that's fair that it isn't your change.

Reply:
Alright, I agree that doing something about the health thing is fine at some point, but it's out of the scope of the pr so I'm going to resolve this for now.